src/PureBlazor.Components/Forms/PureRadioGroup.razor

```diff
@@ -0,0 +1,56 @@
+﻿@using System.Text.RegularExpressions
+@if (Items != null)
+{
+    <fieldset>
+        @if (!String.IsNullOrEmpty(Legend))
+        {            
+            <legend class="text-sm font-medium leading-6 text-gray-900">@Legend</legend>
+        }
+        @foreach (var option in @Items)
+        {
+            @if (!String.IsNullOrEmpty(@option.Title))
+            {
+                <div class="mt-2 space-y-4">
+                    <div class="relative flex items-start">
+                        <div class="absolute flex h-6 items-center">
+                            <input id="@option.Id" name="@Name" value="@option.Id"
+                                   @onchange="@(() => OnClick.InvokeAsync(@option.Id))"
+                                   aria-describedby="privacy-public-description"
```

Comment:
this should point to the description i believe

Reply:
I'm confused, Isn't that how is in the CMS?
![image](https://github.com/pureblazor/components/assets/2971840/8231ff83-9343-47e0-a1e8-1059c3619496)

Reply:
Yes, but it's hard-coded there because it is only used here. This component should be dynamic and change based on what consumers of this component provide for data. Let's chat in a bit and I can explain.

See here for some relevant information:
https://www.w3.org/TR/WCAG20-TECHS/ARIA1.html